### ISSUE TITLE
[BUG] - phpVMS5 cancel.php file has incorrect statement.

### ISSUE THREAD

**Issue by walkerairtrans** [invernyx/smartcars-3-public-api#99]
### Describe the bug

```
if($database->fetch('SELECT id FROM ' . dbPrefix . 'bids WHERE id=? AND user_id=?', array($_POST['bidID'], $pilotID)) === array())
{
    error(404, 'There is no ongoing flight');
}
```
should be this
```
if($database->fetch('SELECT id FROM ' . dbPrefix . 'bids WHERE bidid=? AND pilotid=?', array($_POST['bidID'], $pilotID)) === array())
{
    error(404, 'There is no ongoing flight');
}
```
id and user_id are not valid columns in the phpVMS5 database structure.

### How do you reproduce this bug?

Check the logs of the server -- you will see this called out in the php handler when trying to call the DB query.

### Expected behavior

Shown above.

### API Version

0.3.7

### PHP Version

7.4

### Database & version

10.5 MariaDB

### Webserver version

Apache

### Browser

Chrome

### Handler

phpvms5

**Comment by GenericNerd** [comment]
Are you able to open up a pull request for this, since you already have the code? I want you to have credit for it :)

**Comment by walkerairtrans** [comment]
Yes sir. Just did (I think successfully.) :)

**Comment by GenericNerd** [comment]
Resolved through #100
